tests/test_make_alias.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from pybind11_tests import ConstructorStats
+
+m = pytest.importorskip("pybind11_tests.make_alias")
+
+
+def assert_name(mat):
+    assert m.__name__ == "make_alias"
```

Comment:
Not sure this is an effective test. You would probably need to check the qualname of an object (class) in the module, maybe have another class in there that has not been aliased